ItsZil/neismesk
# Questionnaire functionality

User can answer questions on ad of type "questionnaire". Owner can then open detailed ad page and select his favorite answers.
After he selects a user, an email is sent to the user with a url, where he can enter his details so that the owner could contact him.

Moved owner/admin buttons in ItemViewPage and also added button to access DetailedItemPage

To test out:

1. Answer questions when viewing ad
2. Click "Atsakymai" button
3. Select a user as the "winner"
4. Check email and check if ad status in DB is 2
5. Enter details after clicking url in email
6. Check email again and check if ad status in DB is 3
7. Check if ad does not appear in home page
